micro-rdk/src/common/data_collector.rs

```diff
@@ -45,7 +53,7 @@ impl TryFrom<&Kind> for DataCollectorConfig {
             .unwrap_or(&Kind::NumberValue(DEFAULT_CACHE_SIZE_KB))
             .try_into()?;
         let capacity = (capacity_kb * 1000.0) as usize;
-        if capacity < 1000 {
+        if (capacity < 1000) && !disabled {
```

Comment:
nit: not actually a change from this PR but wouldn't this (without the casting to usize) be the same as `if capacity_kb < 1`?